hudi-notebooks/build.sh

```diff
@@ -43,3 +45,11 @@ docker build \
     -t apachehudi/hive:latest \
     -t apachehudi/hive:"$HIVE_VERSION_TAG" \
     -f "$SCRIPT_DIR"/Dockerfile.hive .
+
+echo "Building Trino Docker image using Trino version: $TRINO_VERSION"
```

Comment:
Should we consider the setup of building the Hudi connector jar from `hudi-trino-plugin` as well as a follow-up?